gemstone/run.sh

```diff
@@ -109,6 +179,18 @@ run_build() {
       ;;
   esac
 
+  # To bypass cached behavior for local build, export GS_TRAVIS_CACHE_ENABLED
+  #   before calling run.sh
+  if [ "${GS_TRAVIS_CACHE_ENABLED}x" = "x" ]; then
```

Comment:
My main thought was that I wanted to be able to test the code locally when `GS_TRAVIS_CACHE_ENABLED=false`, since that is a separate code path in gemstone/run.sh.